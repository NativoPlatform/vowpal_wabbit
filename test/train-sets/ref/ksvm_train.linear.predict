0
0.532215
0.049905
-0.405208
-0.487604
-0.580532
-0.403652
-0.486907
-0.496208
-0.586543
-0.461558
-0.514863
-0.557022
-0.661419
-0.632813
-0.608730
-0.359330
-0.384717
-0.427091
-0.474414
-0.673423
-0.366528
-0.288943
-0.338795
-0.119199
-0.458678
-0.616314
-0.705937
-0.347474
-0.248472
-0.560634
-0.300395
-0.496642
-0.436556
-0.555171
-0.411741
-0.532476
-0.324244
-0.399172
-0.062069
-0.346901
-0.177192
-0.280171
-0.147740
-0.548391
-0.660623
-0.302068
-0.469732
-0.598375
-0.219459
-0.435930
-0.361022
-0.145709
-0.456742
-0.449130
0.302024
-0.553600
-0.608774
-0.472564
-0.331318
-0.341241
-0.217381
0.227918
-0.780847
-0.249760
-0.133497
-0.291213
-0.375556
-0.654420
-0.350120
0.049954
0.081509
-0.191879
-0.119475
-0.003383
0.038429
-0.255158
-0.574866
-0.324819
-0.387747
-0.407099
-0.284335
0.006548
-0.140787
-0.658066
-0.431984
-0.549671
0.006252
-0.211740
-0.452373
-0.398931
-0.367127
0.535623
-0.213845
0.102286
-0.449988
-0.364565
-0.119938
-0.498516
-0.307306
-0.191912
-0.100491
-0.244282
-0.256559
0.050751
-0.230626
-0.215882
-0.359820
-0.299230
-0.167031
0.186938
-0.110400
-0.061144
-0.535305
0.468306
-0.301153
-0.650467
-0.250652
-0.177280
-0.481051
0.011821
0.035631
-0.244157
-0.550356
0.218108
0.385807
-0.373548
0.176403
-0.416168
-0.097865
-0.005806
0.057600
0.094829
-0.348327
0.025725
-0.469431
-0.106318
-0.024182
0.755941
-0.170761
-0.422678
-0.517043
0.555913
0.753465
-0.332527
0.426389
-0.028109
0.689358
-0.856083
-0.335118
0.439359
-0.098466
0.297016
-0.101006
-0.561070
0.274523
0.779732
-0.142178
0.593026
-0.060769
0.801555
-0.434696
-0.477041
0.255845
-0.291906
-0.215789
0.504130
-0.211879
-0.381830
0.688208
0.760717
-0.731383
-0.441206
0.103471
-0.670370
-0.097207
-0.883259
0.740255
0.265271
0.540655
0.112594
0.214028
0.303570
-0.390744
-0.215541
-0.189281
0.288952
-0.400850
-0.233201
0.213064
0.682345
-0.088726
-0.467387
-0.437492
-0.456360
0.004307
0.143748
-0.283180
-0.046762
-0.082755
-0.314598
0.102055
0.275380
-0.495318
-0.509942
-0.679638
0.605376
-0.031532
-0.064217
-0.000307
0.036649
-0.807109
-0.490383
-0.009692
1.379644
-0.746544
-0.397676
-0.395333
-0.387111
-0.385771
-0.087943
-0.588115
-0.531130
-0.228442
0.052496
-0.117043
1.190074
0.192485
-0.451175
-0.684417
-0.660984
-0.599292
-0.823021
-0.026535
0.243273
-0.436865
-0.479339
0.218232
0.920174
0.001353
0.335024
-0.195811
-0.870467
0.490842
-0.023048
0.446589
0.424963
-0.501702
-0.593877
0.530792
