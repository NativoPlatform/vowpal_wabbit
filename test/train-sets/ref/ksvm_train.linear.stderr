using l2 regularization = 1
predictions = ksvm_train.linear.predict
Lambda = 1
Kernel = linear
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.266108 1.532215            2            2.0  -1.0000   0.5322      103
1.044228 0.822348            4            4.0  -1.0000  -0.4052      134
0.922410 0.800592            8            8.0  -1.0000  -0.4869      145
0.907269 0.892127           16           16.0   1.0000  -0.6087       23
0.910573 0.913877           32           32.0  -1.0000  -0.3004       31
0.904033 0.897494           64           64.0  -1.0000  -0.7808       60
0.915012 0.925991          128          128.0   1.0000   0.1764      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.808040
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 242
Number of kernel evaluations = 141483 Number of cache queries = 227710
Total loss = 202.009979
Done freeing model
Done freeing kernel params
Done with finish 
